src/file_utils.cr

```diff
@@ -41,8 +41,8 @@ module FileUtils
   def cmp(filename1 : String, filename2 : String)
     return false unless File.size(filename1) == File.size(filename2)
 
-    File.open(filename1, "rb") do |file1|
-      File.open(filename2, "rb") do |file2|
+    File.open(filename1) do |file1|
```

Comment:
What happened to the binary mode?

Reply:
The binary mode is a no-op, see https://github.com/crystal-lang/crystal/blob/master/src/crystal/system/file.cr#L33

Reply:
Yes, but even if it's no-op today, wouldn't that be needed eventually or for other platforms? Almost every language still allows you to specify binary mode. Probably I'm missing some history lesson here.

Reply:
Binary mode applies only to windows, and it's not implemented there either. I think adding binary/text mode support is a new feature and out of scope for this PR, so I'd rather discuss this elsewhere.